R/misc.R

```diff
@@ -0,0 +1,25 @@
+#' Pass command lines to a shell
+#'
+#' Cross-platform function to pass a command to the shell, using either
+#' \code{\link[base]{system}} or
+#' (Windows-only) \code{\link[base]{shell}}, depending on the operating system.
+#'
+#' @param commandstring String.
+#' One or multiple commandlines as one would enter in the shell.
+#' @param ... Other arguments passed to \code{\link[base]{system}} or
+#' \code{\link[base]{shell}}.
+#'
+#' @inheritParams base::system
```

Comment:
I think this will work because `base::shell` inherits from `base::system`?

Reply:
`system()` is available everywhere. `@inheritParams` just triggers importing the _documentation_ of its arguments, it does not influence the function. Also, `shell()` supports `system()` arguments through `...` (I believe it is a front-end to `system()`).